### ISSUE TITLE
Multi-Site: Cannot localize asset metadata i.e. alt text, caption,...

### ISSUE THREAD

**Issue by peterkelm** [statamic/cms#4191]
## Bug Description
Asset alt text, caption, etc. are overwritten even though the associated blueprint field is localizable. Also, I didn't find documentation on how this could be accomplished outside of the control panel via direct editing of the files...

## How to Reproduce
- Using the Statamic "creek" starter kit
- Convert to multisite and add a second language
- Make the alt text field localizable
- Add an alt text to one of the existing assets and save it
- Change to the other language
- Open the asset again, the alt text from the first language is displayed... Editing it changes it in "both" languages

## Extra Detail
Nothing in particular...

## Environment
Statamic 3.1.35 Pro
Laravel 8.55.0
PHP 7.3.11
statamic/ssg 0.4.0

**Install method (choose one):**
- Starter kit

**Comment by jasonvarga** [comment]
Duplicate of https://github.com/statamic/ideas/issues/223

**Comment by vitalijalbu** [comment]
any updates about this feature guys?
@jasonvarga 
I'm on v5+ and need that so bad...

**Comment by duncanmcclean** [comment]
Sorry, we don't have any updates.

When we build it, it'll be for Statamic 6 though. We're not adding any new features to v5.
